### PR TITLE
Additional entries for Encyclopedia planetary focus game concept page

### PR DESCRIPTION
**Pull request by rrobgill**
Additional text added for Spatial Distortion, Industry, Logistics,
Planet Drive, and Psychic Domination

Signed-off-by: Rob Gill <rrobgill@protonmail.com>

### CODE REVIEW

**Comment by geoffthemedio** [comment]
````
--- default/stringtables/en.txt	2020-06-12 00:22:38.308580015 +0000
+++ /dev/fd/63	2020-06-12 00:22:39.716700129 +0000
@@ -6086,7 +6086,6 @@
 
 When the Psychic Domination Focus is engaged it also reduces the chance that [[predefinedshipdesign SM_PSIONIC_SNOWFLAKE]]s take over your ships.'''
 
-
 STOCKPILE_FOCUS_TITLE
 Stockpile Focus
 
String table is not properly formatted
````
extra blank line?

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -6015,6 +6030,12 @@ The focus of a planet may grant it bonuses or enable effects. A small penalty ap
 
 A planet will default to the inhabiting species' preferred focus if it has one, otherwise it will default to [[metertype METER_INDUSTRY]].'''
 
+DISTORTION_FOCUS_TITLE
+Spatial Distortion Focus
+
+DISTORTION_FOCUS_TEXT
+Available after construction the [[BLD_SPATIAL_DISTORT_GEN]], the properties of each starlane leading to the planet can be manipulated to slow down incoming enemy ships, retarding their progress by 40uu per turn. Slow ships may even be forced back to their origin system.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: "origin" might be misleading... maybe "to the system at the other end of the starlane"? Kinda wordy, though...

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
 
 [[GROWTH_SPECIALS_ENTRY_LIST]]'''
 
+INDUSTRY_FOCUS_TITLE
+Industry Focus
+
+INDUSTRY_FOCUS_TEXT
+The Industry Focus represents a plantey primarily focused on industrial production. [[metertype METER_INDUSTRY]] will receive a bonus which increases as the planetary population increases. This is the default focus for any species that does not have a preferred focus.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: plantey -> planet

"industry means focus on industry" is a bit redundant... maybe "Planets focused on Industry will generate more [[metertype METER_INDUSTRY]] output for their empire, which is used for producing ships and buildings, or may be stockpiled for later use. Industry focus increases output more for higher population planets."

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
 Influence Focus
 
 INFLUENCE_FOCUS_TEXT
 Influence isn't implemented yet.
 
+LOGISTICS_FOCUS_TITLE
+Logistics Focus
+
+LOGISTICS_FOCUS_TEXT
+By focusing on Logistics, a planet is able to vastly extend its supply. The [[metertype METER_SUPPLY]] will be increased by a factor of three.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Logistics focus increases the [[metertype METER_SUPPLY]] of the planet is increased by 3. This extends the planet's empire's supply network from the planet by 3 starlane jumps.

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
+
+PLANETDRIVE_FOCUS_TITLE
+Planet Drive Focus
+
+PLANETDRIVE_FOCUS_TEXT
+After a [[BLD_PLANET_DRIVE]] has been constructed, and a [[buildingtype BLD_PLANET_BEACON]] or [[shippart SP_PLANET_BEACON]] is in an adjacent system, selecting the [[PLANETDRIVE_FOCUS_TITLE]] allows the planet to move between star systems.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: probably [[buildingtype BLD_PLANET_DRIVE]]

allows the planet to move -> will move the planet

File: default/stringtables/en.txt [hunk 5]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
 Protection Focus
 
 PROTECTION_FOCUS_TEXT
 The Protection focus represents a planet wholly focused on defending itself. [[metertype METER_TROOPS]], Planetary [[metertype METER_DEFENSE]], and Planetary [[metertype METER_SHIELD]] are doubled.
 
+PSYCHIC_FOCUS_TITLE
+Psychic Domination Focus
+
+PSYCHIC_FOCUS_TEXT
+'''The [[PSYCHIC_FOCUS_TITLE]] concentrates the mental force exerted by the entire population to dominate the thoughts of non-telepathic crews of ships in the same system.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: ships -> enemy empire ships

Comment (geoffthemedio) [reply default/stringtables/en.txt]: (unless it also affects monsters?)

File: default/stringtables/en.txt [hunk 6]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
+
+Each turn there is a 10% chance of taking control of of their ships unless the enemy empire also posesses the technology [[LRN_PSY_DOM]].
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: ...10% change of taking control of each ship, unless the...

File: default/stringtables/en.txt [hunk 7]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
+
+When the Psychic Domination Focus is engaged it also reduces the chance that [[predefinedshipdesign SM_PSIONIC_SNOWFLAKE]]s take over your ships.'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: ...engaged, it also...

...[[predefinedshipdesign SM_PSIONIC_SNOWFLAKE]]s will take control of ships controlled by the planet's empire.

File: default/stringtables/en.txt [hunk 8]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
 
 [[GROWTH_SPECIALS_ENTRY_LIST]]'''
 
+INDUSTRY_FOCUS_TITLE
+Industry Focus
+
+INDUSTRY_FOCUS_TEXT
+Planets focussed on Industry will generate more [[metertype METER_INDUSTRY]] will output for their empire, which is used for producing ships and buildings, or may be stockpiled for later use. Industry focus increases output more for higher population planets.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: focused

File: default/stringtables/en.txt [hunk 9]
```diff
@@ -6015,6 +6030,12 @@ The focus of a planet may grant it bonuses or enable effects. A small penalty ap
 
 A planet will default to the inhabiting species' preferred focus if it has one, otherwise it will default to [[metertype METER_INDUSTRY]].'''
 
+DISTORTION_FOCUS_TITLE
+Spatial Distortion Focus
+
+DISTORTION_FOCUS_TEXT
+Manipulates starlanes leading to the planet slowing the progress of enemy ships by 40uu per turn. Slow ships may even be forced back to the system at the far end of the starlane. Available after constructing a [[buildingtype BLD_SPATIAL_DISTORT_GEN]].
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: 40 uu

File: default/stringtables/en.txt [hunk 10]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
 Influence Focus
 
 INFLUENCE_FOCUS_TEXT
 Influence isn't implemented yet.
 
+LOGISTICS_FOCUS_TITLE
+Logistics Focus
+
+LOGISTICS_FOCUS_TEXT
+Logistics focus increases the [[metertype METER_SUPPLY]] of the planet is increased by 3. This extends the planet's empire's supply network from the planet by 3 starlane jumps.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: Style inconsistent: `Planets focused on Industry...` vs. `Logistics focus increases...`

File: default/stringtables/en.txt [hunk 11]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
+
+PLANETDRIVE_FOCUS_TITLE
+Planet Drive Focus
+
+PLANETDRIVE_FOCUS_TEXT
+After a [[buildingtype BLD_PLANET_DRIVE]] has been constructed, and a [[buildingtype BLD_PLANET_BEACON]] or [[shippart SP_PLANET_BEACON]] is in an adjacent system, selecting the [[PLANETDRIVE_FOCUS_TITLE]] will move the planet to move between star systems.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: `will move the planet to move` -> `will move the planet`

File: default/stringtables/en.txt [hunk 12]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
 Protection Focus
 
 PROTECTION_FOCUS_TEXT
 The Protection focus represents a planet wholly focused on defending itself. [[metertype METER_TROOPS]], Planetary [[metertype METER_DEFENSE]], and Planetary [[metertype METER_SHIELD]] are doubled.
 
+PSYCHIC_FOCUS_TITLE
+Psychic Domination Focus
+
+PSYCHIC_FOCUS_TEXT
+'''The [[PSYCHIC_FOCUS_TITLE]] concentrates the mental force exerted by the entire population to dominate the thoughts of non-telepathic crews of enemy empire ships in the same system.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: `exerted by the entire` -> `of the`

File: default/stringtables/en.txt [hunk 13]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
+
+Each turn there is a 10% chance of taking control of each ship, unless the enemy empire also posesses the technology [[LRN_PSY_DOM]].
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: `Each turn, there is`

File: default/stringtables/en.txt [hunk 14]
```diff
@@ -6025,18 +6046,46 @@ A planet focused on Growth extends those benefits to other planets connected to
...
+
+When the Psychic Domination Focus is engaged it also reduces the chance that [[predefinedshipdesign SM_PSIONIC_SNOWFLAKE]]s will take control of ships controlled by the planet's empire.'''
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: `is engaged, it also`

Is this effect in system only, or for all ships of the empire anywhere if any planet has the focus?

**Comment by Vezzra** [comment]
FYI: Had to resolve a merge conflict in `en.txt` when cherry picking to the release branch. Hopefully got it right.

**Pull request merged**